vumi/persist/riak_manager.py

```diff
@@ -293,16 +293,10 @@ def _search_iteration(self, bucket, query, rows, start):
 
     def real_search(self, modelcls, query, rows=None, start=None):
         rows = 1000 if rows is None else rows
+        start = 0 if start is None else start
```

Comment:
> I don't think these other places exist? And if there are a couple of places we can fix them when we find them?

How will we find them if we don't raise an exception? 

> The cost of raising the exception is having yet another ugly function in the code base that we have to clean up later.

The cost of not raising the exception is potentially introducing subtle data corruption bugs into existing code. Having been on the receiving end of that kind of change more than once, I'll take the ugly function every time.